android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -87,13 +87,13 @@ protected void onResume() {
         //setting the grid layout to cut-off white space in tablet view
         DisplayMetrics displayMetrics = this.getResources().getDisplayMetrics();
         float width = displayMetrics.widthPixels / displayMetrics.density;
-        int spanCount = (int) (width/250.00);
+        int spanCount = (int) (width / 250.00);
```

Comment:
@arp95 This happens when I auto indent and putting spaces between the operatorand operand is encouraged. Read more about it here https://google.github.io/styleguide/javaguide.html . Do you also see other places where there is space already?